index.html

```diff
@@ -11,51 +11,187 @@
-                </li>
-            </ul>
-        </div>
-
-
-    </section>
-
-    <section class="my-projects">
-        <div class="down-arrow">
-            <img src="./images/arrow.png">
-        </div>
-    </section>
+
+        <header>
+            <h2 class="logo">Welcome</h2>
+            <nav>
+                <i class="fas fa-bars"></i>
+            </nav>
+        </header>
+
+        <section class="intro">
+            <div class="content">
+                <h1 class="about">Hey there. I’m <br>Lynette <br /><span>I’m a software<br/>developer</span></h1>
+                <p class="introduction"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.
+                </p>
+            
+
+                <ul class="page-links">
+                    <li>
+                        <a class="icon-list" href="https://github.com/iLynette"><i class="fab fa-github"></i></a>
+                    </li>
+
+                    <li>
+                        <a class="icon-list" href="https://www.linkedin.com/in/lynette-acholah/"><i class="fab fa-linkedin"></i></a>
+                    </li>
+
+                    <li>
+                        <a class="icon-list" href="https://github.com/iLynette"><i class="fab fa-angellist"></i></a>
+                    </li>
+
+                    <li>
+                        <a class="icon-list" href="https://twitter.com/acholah_lynette"><i class="fab fa-twitter"></i></a>
+                    </li>
+
+                    <li>
+                        <a class="icon-list" href="https://medium.com/@acholahlynette111"><i class="fab fa-medium"></i></a>
+                    </li>
+                </ul>
+            </div>
+
+
+        </section>
+
+        <section class="my-projects">
+            <div class="down-arrow">
+                <img src="./images/arrow.png" alt="down-arrow">
+            </div>
+
+            <div class="title">
+                <h2>My Recent Works</h2>
+            </div>
+
+            <div class="line">
+                <img src="./images/greenline.png" alt="greenline">
+            </div>
+
+            <div class="project-container">
+                <div class="single-project">
+                    <img src="./images/placeholder.png" alt="placeholder">
+                    <div class="project-title">
+                        <h4>Multi-Post Stories Gain+Glory</h4>
+                    </div>
+                    <ul class="tech-stack">
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+
+                    <div class="btn">
+                        <button type="button">See Project</button>
+                    </div>
```

Comment:
- [ ] Excellent work following the template design 👍 but there's still a small issue with your buttons. You are to implement the following button interactions as shown in the template to all your buttons 💯 
![h1](https://user-images.githubusercontent.com/85113567/148634007-d2abb62a-a5f8-4426-90eb-779f019330ea.jpg)